man/systemd-resolved.service.xml

```diff
@@ -116,28 +118,38 @@
     according to the following rules:</para>
 
     <itemizedlist>
-      <listitem><para>Lookups for the special hostname <literal>localhost</literal> are never routed to the
-      network. (A few other, special domains are handled the same way.)</para></listitem>
+      <listitem><para>Names for which synethetic records as generated (as listed in the previous section) are
+      never routed to the network and a reply is sent immediately. In particular this means that lookups for
+      <literal>localhost</literal> are never routed to the network.</para></listitem>
 
       <listitem><para>Single-label names are routed to all local interfaces capable of IP multicasting, using
       the LLMNR protocol. Lookups for IPv4 addresses are only sent via LLMNR on IPv4, and lookups for IPv6
       addresses are only sent via LLMNR on IPv6. Lookups for the locally configured host name and the
       <literal>_gateway</literal> host name are never routed to LLMNR.</para></listitem>
 
       <listitem><para>Multi-label names with the domain suffix <literal>.local</literal> are routed to all
-      local interfaces capable of IP multicasting, using the MulticastDNS protocol. As with LLMNR IPv4
+      local interfaces capable of IP multicasting, using the MulticastDNS protocol. As with LLMNR, IPv4
       address lookups are sent via IPv4 and IPv6 address lookups are sent via IPv6.</para></listitem>
 
+      <listitem><para>Single-label names are routed to any interfaces which define search domains, suffixed
```

Comment:
`to any interfaces` -> `to any interface`

Reply:
OK, I completely rewrote the paragraph. It's now full of repetitions, but I don't see a way to remove that without losing meaning.